package.json

```diff
@@ -50,6 +50,7 @@
     "es",
     "lib",
     "umd",
+    "orbit-icons-font",
```

Comment:
I would also publish the generated zip file in root folder. It would be nice to have it linked on orbit.kiwi, the folder itself can be useful for React Native etc.